server/repository/src/db_diesel/document_registry_config.rs

```diff
@@ -58,19 +79,24 @@ pub struct EventScheduleInConfig {
 pub struct EventScheduleConfig {
     /// Field with the scheduled base time.
     /// If not specified the document base time is used.
```

Comment:
document base time ? (document start time ?), or can we configure document base time somewhere ?

Reply:
I added a comment about it. Think it would be nice to be able to configure it. You said you had some other ideas to improve the config? I also had some more idea about it, lets sit together and go over it...

Also realized that the programs event are not really program specific, but not sure if it is worth to rename them to "document events"...

Reply:
>  I also had some more idea about it, lets sit together and go over it...

Great, I've been adding my thoughts to a note, I am not sure I'll have the time to write it up today, and I am away tomorrow. Maybe for Monday or Tuesday ?